app/src/main/java/io/gnosis/safe/ui/safe/selection/SafeSelectionViewModel.kt

```diff
@@ -35,25 +35,28 @@ class SafeSelectionViewModel @Inject constructor(
 
     fun loadSafes() {
         safeLaunch {
-            val safes = safeRepository.getSafes()
             activeSafe = safeRepository.getActiveSafe()
+            val safes = safeRepository.getSafes().filter { it != activeSafe }.reversed()
```

Comment:
this looks a bit hard to understand what's happening. Maybe it would be good to generate all the safes in the same place in a more declarative fashion? 
```kotlin
with(items){
    val activeSafe = safeRespository.getActiveSafe()
    activeSafe?.let(::add)
    addAll(safeRepository.getSafes().filter { it != activeSafe }.reversed()
}
```